language_names/manual-additions.tsv

```diff
@@ -0,0 +1,150 @@
+lg	inLg	name
```

Comment:
Could you sort all the manual files?

Reply:
Do you mean alphabetically?

Reply:
Yes. In fact, I'd like all the files to be sorted (and the scraped ones should be sorted by the script). This will make diffs much easier to see in the future.

Reply:
Ohh nice idea. I will change these files.